input/fsh/VRCL/harmonized-profiles/ObservationCodedRaceAndEthnicity.fsh

```diff
@@ -8,7 +8,7 @@ This profile is only for use in a Vital Records Exchange. Systems looking to exc
 
 Note:  NCHS stopped providing RaceRecode40 data in 2022."
 * insert RequireMetaProfile(Profile: CodedRaceAndEthnicityNew)
-* code = ObservationsCS#codedraceandethnicity
+* code from ValueSetCodedRaceAndEthnicityPersonVitalRecords (required) //  code = ObservationsCS#codedraceandethnicity
 * value[x] 0..0
```

Comment:
There were some minor differences between this version and the published version. Here there is:
- a constraint on focus 0..* --> 0..1 and a constraint on referenced profiles
- no effective[x]

Reply:
What should it be?